julia-mode.el

```diff
@@ -199,6 +199,12 @@
   "for +.*[^
 ].* \\(in\\|∈\\)\\(\\s-\\|$\\)+")
 
+(defconst julia-ternary-regex
+  " +\\(\\?\\)[
```

Comment:
I personally prefer using `rx` as it's easier to read, but leaving it as this looks ok.

Reply:
I agree. By the time I realized it was hairy enough that `rx` would have improved readability, it was mostly complete and didn't seem worth rewriting :man_shrugging:. If you review bdb667e that I just pushed you'll find that I also wrote the regex manually there, but that one was intentional since it's very similar to the regex immediately above it, so it's simpler to see the differences if they're both written in the same form.

I'm happy to rewrite both using `rx` though if there's a strong consensus that it's worthwhile to do so.

Reply:
I am fine with it as is.